.circleci/config.yml

```diff
@@ -215,33 +215,29 @@ jobs:
             unzip -q "$CIRCLE_ARTIFACTS/content_test.zip" -d "test_content_dir"
             zip -j "$CIRCLE_ARTIFACTS/all_content.zip" test_content_dir/*
             rm -r test_content_dir
-      - store_artifacts:
-          path: artifacts
-          destination: artifacts
       - run:
           name: Run Unit Testing and Lint
           when: always
+          no_output_timeout: 5h
           command: |
             if [ -n "${INSTANCE_TESTS}" ] || [ $CIRCLE_NODE_INDEX -ne 1 ] ;
               then
                 echo "Skipping - unit tests are running only in container number 1"
                 exit 0
             fi
-            echo "demisto-sdk version: $(demisto-sdk --version)"
-            echo "mypy version: $(mypy --version)"
-            echo "flake8 py2 version: $(python2 -m flake8 --version)"
-            echo "flake8 py3 version: $(python3 -m flake8 --version)"
-            echo "bandit py2 version: $(python2 -m bandit --version 2>&1)"
-            echo "bandit py3 version: $(python3 -m bandit --version 2>&1)"
-            echo "vulture py2 version: $(python2 -m vulture --version) 2>&1"
-            echo "vulture py3 version: $(python3 -m vulture --version) 2>&1"
```

Comment:
Why did you delete all these lines? Please bring them back. It is important when debugging to understand the exact version being used of each lint component.

Reply:
I added them back.